src/main.py

```diff
@@ -113,8 +127,36 @@ def transcribe():
         finally:
             tempFile.close()
 
+@app.route('/v1/download/<job_id>', methods=['GET'])
+def download(job_id):
+    output = request.args.get("output", DEFAULT_OUTPUT)
+
+    try:
+        job = Job.fetch(job_id, connection=conn)
+    except NoSuchJobError:
+        return "No such job", 404
+    
+    if job.is_finished:
+        if output == "txt":
+            return result["text"]
+        if output == "json":
+            return result        
+        if output == "vtt":
+            return generate_vtt(result["segments"]), 200, {'Content-Type': 'text/vtt', 'Content-Disposition': 'attachment; filename=transcription.vtt'}
+        if output == "srt":
+            return generate_srt(result["segments"]), 200, {'Content-Type': 'text/plain', 'Content-Disposition': 'attachment; filename=transcription.srt'}
```

Comment:
I may be missing something, but I dont see where `result` is defined? Does not look like it will work?

Reply:
I don't think you missed anything! I rather missed to remove this bit. Replacing with HTTP 201 and confirmation that it's added to the queue!

Reply:
Hum… this is the download endpoint. Don’t we want this to return the data with a 200OK? We should definitely keep the result in my opinion

Reply:
We'll need that code again in the code to send e-mail, though, I think!